src/main/java/com/karankumar/bookproject/ui/goal/ReadingGoalView.java

```diff
@@ -107,7 +107,7 @@ private void updateSetGoalText() {
         setGoalButton.setText(UPDATE_GOAL);
     }
 
-    private void saveGoal(ReadingGoalForm.SaveEvent event) {
+    private void saveGoal(ReadingGoalForm.SaveGoalEvent event) {
```

Comment:
`ReadingGoalForm` does not have an inner class called `SaveGoalEvent`. Could you please remove `ReadingGoalForm.` here?